src/asm/symbol.c

```diff
@@ -24,8 +24,24 @@ char SavedTIME[256];
 char SavedDATE[256];
 char SavedTIMESTAMP_ISO8601_LOCAL[256];
 char SavedTIMESTAMP_ISO8601_UTC[256];
+char SavedDAY[3];
+char SavedMONTH[3];
+char SavedYEAR[5];
+char SavedHOUR[3];
+char SavedMINUTE[3];
+char SavedSECOND[3];
 bool exportall;
 
+void helper_RemoveLeadingZeros(char * string){
+	char * new_beginning = string;
+
+	while(new_beginning[0] == '0'){
```

Comment:
This will eat up all zeros if the number is "00" and leave an empty string. It's not like that could happen here, but if it's going to be a generic helper it should work in all cases. Something like this should be good enough:
```
void helper(char * s)
{
	int i = 0;

	while(s[i] == '0') i++;

	if (i == 0) break;

	if (s[i] == '\0') i--;

	memmove(s, &(s[i]), strlen(&(s[i])) + 1);
}
```